index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Medical core</title>
+</head>
+<body>
+    <div class="medical-header__container">
```

Comment:
`medical-header__container` is an element without a parent block `medical-header`

Reply:
a block comes before an element